apidoc/Titanium/UI/ListView.yml

```diff
@@ -397,6 +397,62 @@ events:
         summary: false. This event does not bubble.
         type: Boolean
 
+  - name: scrollstart
+    summary: Fires when the listview starts scrolling by user interaction. Calling the `scrollTo` methods will not fire this event
+    description: |
+        This event will fire when the user starts scrolling the listview. Use the `firstVisibleSectionIndex` and 
+        the `firstVisibleItemIndex` to determine the position of the listview.
+    platforms: [iphone, ipad, android]
+    since: 4.1.0
+    properties: 
+      - name: visibleItemCount
+        summary: The number of visible items in the listView when the event fires
+        type: Number
+
+      - name: firstVisibleItem
+        summary: The first visible item in the listView when the event fires, this item might not be fully visible
```

Comment:
These are two separate sentences.  You can either change the comma to a semicolon, or change the comma to a period and capitalize 'this'.

Repeat for firstVisibleItemIndex and the same for the scroll end event.